### ISSUE TITLE
NextJs example doesn't import @manahippo/aptos-wallet-adapter

### ISSUE THREAD

**Issue by kingaptos** [hippospace/aptos-wallet-adapter#79]
I suppose you have it globally installed, so the problem only happens when you try to run this externally. 

The fix is simple, add `@manahippo/aptos-wallet-adapter` to `package.json`.

<img width="1035" alt="Screen Shot 2022-10-23 at 4 59 17 PM" src="https://user-images.githubusercontent.com/111548547/197420197-55f4b716-e153-49be-b649-df572069ccb5.png">

**Comment by quagliero** [comment]
Yeah it looks like this needs changing from `file:` to the actual package: https://github.com/hippospace/aptos-wallet-adapter/blob/main/packages/wallet-nextjs/package.json#L13
